.github/workflows/lint-sample.yml

```diff
@@ -19,15 +26,12 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
+      - name: Prepare ${{ matrix.folder }}
+        working-directory: ${{ matrix.folder }}
+        run: go mod tidy
+      - name: Check linter configuration
+        working-directory: ${{ matrix.folder }}
+        run: make lint-config
       - name: Run linter
-        uses: golangci/golangci-lint-action@v6
-        with:
-          version: v1.59
-          working-directory: testdata/project-v4
-          args: --config .golangci.yml ./...
-      - name: Run linter
-        uses: golangci/golangci-lint-action@v6
-        with:
-          version: v1.59
-          working-directory: testdata/project-v4-with-plugins
-          args: --config .golangci.yml ./...
```

Comment:
HI @mateusoliveira43 

I gave an wrong direction here sorry that is 100% my fault
We will need 


```
      - name: Run linter
        uses: golangci/golangci-lint-action@v6
        with:
          version: v1.59
          working-directory: testdata/project-v4-with-plugins
          args: --config .golangci.yml ./...
```

Otheriwse, if we broke the lint in the project that we scaffold we will not see GitHub comments